src/modules/disable_vod_autoplay/index.js

```diff
@@ -0,0 +1,21 @@
+const $ = require('jquery');
+const settings = require('../../settings');
+const watcher = require('../../watcher');
+
+class DisableVodAutoplay {
+    constructor() {
+        settings.add({
+            id: 'disableVodAutoplay',
+            name: 'Disable Vod Autoplay',
```

Comment:
this setting is not self explanatory. i had to read the code to understand what "disable vod autoplay" meant. it should be something named around disabling autoplay of recommended videos